DNA_Library.py

```diff
@@ -0,0 +1,111 @@
+DNAtranscript = {"A": "U", "T": "A", "C": "G", "G": "C",
+                 "a": "u", "t": "a", "c": "g", "g": "c"}
+DNAcompl = {"A": "T", "T": "A", "C": "G", "G": "C",
+            "a": "t", "t": "a", "c": "g", "g": "c"}
+RNAcompl = {"A": "U", "U": "A", "C": "G", "G": "C",
+            "a": "u", "u": "a", "c": "g", "g": "c"}

```

Comment:
Такие глобальные константы, используемые в разных частях программы, принято называть полностью заглавными буквами
```suggestion
DNA_TRANSCRIPT = {"A": "U", "T": "A", "C": "G", "G": "C",
                 "a": "u", "t": "a", "c": "g", "g": "c"}
DNA_COMPL = {"A": "T", "T": "A", "C": "G", "G": "C",
            "a": "t", "t": "a", "c": "g", "g": "c"}
RNA_COMPL = {"A": "U", "U": "A", "C": "G", "G": "C",
            "a": "u", "u": "a", "c": "g", "g": "c"}

```